docs/release-notes.txt

```diff
@@ -8,6 +8,52 @@
  Version 0.14
 **************
 
+Version 0.14.6
+==============
+
+**Release Date:** April 1, 2021
+
+**New Features**
+
+-  API: Add a new endpoint to delete experiments. This endpoint is only
```

Comment:
non-blocking: Typically we say "REST API" to distinguish from our trial apis, etc.